packages/suite/src/components/guide/Header.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import * as guideActions from '@suite-actions/guideActions';
+import { useActions } from '@suite-hooks';
+import styled, { css } from 'styled-components';
+import { Icon, variables, useTheme } from '@trezor/components';
+
+const HeaderWrapper = styled.div<{ noLabel?: boolean }>`
+    display: flex;
+    align-items: center;
+    padding: 22px 22px 27px 22px;
+    ${props =>
+        props.noLabel &&
+        css`
+            justify-content: space-between;
+        `}
+`;
+
+const ActionButton = styled.button`
+    border: 0;
+    background: none;
+    left: auto;
+    cursor: pointer;
+`;
+
+const MainLabel = styled.div`
+    font-size: ${variables.FONT_SIZE.BIG};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    text-align: left;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    width: 100%;
+`;
+
+const Label = styled.div`
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.DEMI_BOLD};
+    text-align: center;
+    color: ${props => props.theme.TYPE_DARK_GREY};
+    padding: 0 15px;
+    width: 100%;
+`;
+
+interface Props {
+    back?: () => void;
+    label?: string | JSX.Element;
+}
+
+const Header = ({ back = undefined, label = undefined }: Props) => {
```

Comment:
has the explicit definition of default values as `undefined` any specific reason?
```suggestion
const Header = ({ back, label }: Props) => {
```